index.html

```diff
@@ -1,13 +1,211 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Inbound Media | Home </title>
   <link rel="stylesheet" href="/dist/css/styles.css">
+
+  <script src="https://kit.fontawesome.com/03d5a1f99a.js" crossorigin="anonymous"></script>
 </head>
-<body>
-  <h1>Hello there</h1>
+
+
+<body class="bg-gray">
+  <!--! Login -->
+  <div class="container mt-5 pt-5 d-flex flex-column flex-lg-row justify-content-evenly">
+    <!--! heading -->
+    <div class="text-center text-lg-start mt-0 pt-0 mt-lg-5 pt-lg-5">
+      <a href="/feed.html"><button class="btn btn-primary w-50 my-1 text-light">Back to home </button></a>
+      <h1 class="text-primary fw-bold fs-0">Inbound Media</h1>
+      <p class="w-75 mx-auto fs-4 mx-lg-0">Inbound Media helps you connect and share with the people in your life.</p>
+      
+    </div>
+    <!-- !form card -->
+    <div style="max-width: 28rem; width: 100%">
+      <!-- !login form -->
+      <!-- !first was form tag -->
+      <div class="bg-white shadow rounded p-3 input-group-lg">
+      <form>
+        <div class="form-group">
+          <label for="email">Email</label>
+          <input pattern="^[A-Za-z0-9._+-]+@[A-Za-z0-9.-]+\.[a-z]{2,4}"   id="email" required   type="email"  class="form-control"   title="Must be a valid email address" 
+           placeholder="name@example.com or phone number"/>
+      
+          </div>
+          
+       
+       
+          <div class="form-group">
+              <label for="password" class="form-control">Password</label>
+              <input title=" Must be at least 8 characters" pattern="[a-zA-Z0-9]{8,}"  class="form-control" required type="text" id="password" />
+                 <p class="alert alert-danger"> Must be at least 8 characters</p>
+          </div>
+     
+      
+       
+          <a href="/feed.html"><button class="btn btn-primary w-100 my-3 text-light">Log In</button></a>
```

Comment:
```suggestion
          <button class="btn btn-primary w-100 my-3 text-light">Log In</button>
```

If you add a form action, you don't need the extra <a> tag. It's also invalid to nest a button inside an anchor tag or vice versa.